### ISSUE TITLE
Hyperparameters for pre-trained models

### ISSUE THREAD

**Issue by ysharma1126** [facebookresearch/adversarial_image_defenses#3]
Thank you very much for providing ImageNet models trained on quilting and TVM. Were the hyperparameters for the transformations the same as presented in the paper?
"We fixed the hyperparameters of our defenses in all experiments: specifically, we set pixel dropout
probability p= 0.5 and the regularization parameter of the total variation minimizer λTV = 0.03. We
use a quilting patch size of 5×5 and a database of 1, 000, 000 patches that were randomly selected
from the ImageNet training set."
If not, what were the hyperparameters?

**Comment by mayankrana** [comment]
Yes, The transformation parameters used in generating models are same as what we have described in paper.
